assignments/prototypes.js

```diff
@@ -15,13 +15,29 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
+const GameObject (){
+  this.createdAt: "Date()",
+  this.name: ,
+  this.dimensions: ,
+  destroy(){
```

Comment:
To create a method, you would want to use syntax like this:
`destroy: function() {}`